website/routes.py

```diff
@@ -183,8 +183,7 @@ def make_url_map(app):
 
         Rule('/about/', 'get', website_views.redirect_about, json_renderer,),
         Rule('/howosfworks/', 'get', website_views.redirect_howosfworks, json_renderer,),
-        Rule('/faq/', 'get', {}, OsfWebRenderer('public/pages/faq.mako')),
-        Rule('/getting-started/', 'get', {}, OsfWebRenderer('public/pages/getting_started.mako')),
```

Comment:
I was under the belief that we try to avoid removing routes, instead maybe redirecting to the new support page. Thoughts,  @sloria?